sdk/core/azure-core-management/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.3.0 (2021-06-07)
 
+
```

Comment:
No changes here, I think we can remove the extra line here

Reply:
file reverted to master